runtime/ftplugin/vim.vim

```diff
@@ -15,6 +15,8 @@ let b:did_ftplugin = 1
 let s:cpo_save = &cpo
 set cpo&vim
 
+compiler vimdoc
+
```

Comment:
remove this if really like to keep vimdoc as a compiler.

Reply:
OK, we can use linter.